tests/examples/source/devfiles/nodejs/README.md

```diff
@@ -0,0 +1,18 @@
+Template from https://github.com/che-samples/web-nodejs-sample
```

Comment:
Maybe rather than storing this in the odo repo we just do a git clone in our tests? I think some of the other odo integration tests clone repos

Reply:
git cloning it right now